src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -0,0 +1,253 @@
+<template>
+  <v-card flat id="people-roles">
+
+    <ConfirmRemoveDialog
+      :dialog="dialog"
+      attach="#people-roles"
+      @confirm="emitRemovePerson(personId)"
+      @exit="dialog = false"
+    />
+
+    <!-- Summary Section -->
+    <div id="people-roles-summary" v-if="isSummary">
+      <!-- Summary Header -->
+      <div class="people-roles-summary-header" >
+        <v-icon>mdi-account</v-icon>
+        <label class="people-roles-title"><strong>People and Roles</strong></label>
+      </div>
+
+      <!-- Summary Warning -->
+      <div v-if="showErrorSummary" class="people-roles-invalid-message">
+        <span>
+          <v-icon color="#1976d2">mdi-information-outline</v-icon>
+          This step is not complete.
+          <router-link id="router-link" :to="{ path: '/add-people-roles', query: { showErrors: true } }">
+            Return to this step to complete it.
+          </router-link>
+        </span>
+      </div>
+    </div>
+
+    <!-- List Display Section -->
+    <div id="people-roles-list">
+      <!-- List Headers -->
+      <v-row class="people-roles-header list-item__subtitle" no-gutters>
+        <v-col v-for="(title, index) in tableHeaders" :key="index">
+          <span>{{title}}</span>
+        </v-col>
+        <!-- Spacer Column For Actions -->
+        <v-col sm="1" v-if="!isSummary"></v-col>
+      </v-row>
+
+      <!-- List Content -->
+      <v-row
+        class="people-roles-content"
+        :class="{ 'list-item__subtitle': !isSummary }"
+        v-for="(person, index) in personList"
+        :key="index"
+        no-gutters>
+        <v-col class="text-truncate">
+          <v-tooltip top :disabled="formatName(person).length < 25" color="primary">
+            <template v-slot:activator="{ on }">
+              <span v-on="on" class="people-roles-title"><strong>{{formatName(person)}}</strong></span>
+            </template>
+            <span>{{formatName(person)}}</span>
+          </v-tooltip>
+        </v-col>
+        <v-col>
+          <base-address class="peoples-roles-mailing-address" :address="person.address.mailingAddress" />
+        </v-col>
+        <v-col>
+          <p v-if="isSame(person.address.mailingAddress, person.address.deliveryAddress)"
+            class="peoples-roles-delivery-address">Same as Mailing Address
+          </p>
+          <base-address v-else class="peoples-roles-delivery-address" :address="person.address.deliveryAddress"/>
+        </v-col>
+        <v-col>
+          <div v-if="person.roles.length>0">
+            <v-col v-for="(role, index) in person.roles" :key="index" class="col-roles">
+              <span>{{role}}</span>
+            </v-col>
+          </div>
+          <div v-else>
+             <v-icon color="rgb(232, 156, 48)" small>mdi-alert</v-icon>
```

Comment:
Check if there is named colour you can use.

Reply:
Fixed!